test/statvar_ranking/missing_Earth_country_rankings.json

```diff
@@ -634,6 +634,13 @@
     ],
     "relatedChart": {}
   },
+  {
```

Comment:
I think this is OK, and because US census is now in a different import-group than india census.

https://datacommons.org/ranking/Count_Person_BelowPovertyLevelInThePast12Months/Country/Earth
vs.
https://autopush.datacommons.org/ranking/Count_Person_BelowPovertyLevelInThePast12Months/Country/Earth

Reply:
But the SV summary suggests we lost 1 place

https://screenshot.googleplex.com/AkxyKNV6WNr2VkQ vs. https://screenshot.googleplex.com/4fwau9Q3NMJewcb

Is that perhaps a mixer merging issue for SVSummary cache?  @shifucun